assessment-api/assessment-actors/src/main/scala/org/sunbird/utils/RequestUtil.scala

```diff
@@ -12,6 +12,7 @@ import scala.collection.JavaConversions._
 object RequestUtil {
 
 	private val SYSTEM_UPDATE_ALLOWED_CONTENT_STATUS = List("Live", "Unlisted")
+	val suggestFrameworks = if (Platform.config.hasPath("question_list_limit")) Platform.config.getInt("question_list_limit") else 20
```

Comment:
What is this variable(suggestFrameworks). It is to define the max limit for question list API right?
Also, the configuration variable list not in the right format (question_list_limit). We use nested structure. Please check other configuration variables once.